config/sync/config_split.config_split.non_local.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 9fb848c4-7570-4a07-9145-2142772e09c7
+langcode: en
+status: true
+dependencies: {  }
+id: non_local
+label: Non-Local
+description: 'Non-local environments, ie, in Cloud.gov.'
+weight: 0
+stackable: false
+no_patching: false
+storage: folder
+folder: ../config/non_local
+module:
+  s3fs: 0
```

Comment:
Is there an s3-mock container people can use in the locals?

Reply:
Split off to https://cm-jira.usa.gov/browse/DIGITAL-181 as we discussed.